cellprofiler/modules/definegrid.py

```diff
@@ -1,7 +1,10 @@
 # coding=utf-8
 
 """
-**Define Grid** produces a grid of desired specifications either
+DefineGrid
+==========
+
+**DefineGrid** produces a grid of desired specifications either
```

Comment:
I don't think the part about the DisplayGridInfo module coming soon (L16-18) is true anymore, plus isn't that function just now part of DefineGrid?  Can you delete that sentence?